Library/Dungeon Fantasy RPG/Classes/Barbarian.gct

```diff
@@ -2065,6 +2117,15 @@
 										"points": -10
 									}
 								},
+								{
+									"id": "15608554-8633-4b15-8e27-a821ccbd640b",
+									"type": "trait_container",
+									"open": true,
+									"name": "Trait Container",
+									"calc": {
+										"points": 0
+									}
+								},
```

Comment:
What's the point of this empty trait container?

Reply:
Typo, fixed